docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -492,7 +492,7 @@ As secret name enter *hopsworks/project/[MY_HOPSWORKS_PROJECT]/role/[MY_SAGEMAKE
     :scale: 20 %
     :figclass: align-center
 
-**Granting access to the secret to the SageMaker notebook role**
+**(Alternative 1) Granting access to the secret to the SageMaker notebook role**
```

Comment:
@steffengr numbering of Alternatives is confusing since they belong to the same section. It's not clear to what action the `Granting access to the secret to the SageMaker notebook role*` is an alternative for.

Reply:
Done

Reply:
I'm not sure how to structure that better in the given document. Suggestion are welcome.